connectors/cache/fallback.go

```diff
@@ -60,6 +60,24 @@ func WithSkipWriteInvalidateEntities(entities ...dosa.DomainObject) Options {
 	}
 }
 
+// Context key used by SetCacheableEndpoints
+var (
+	// ContextEndpoint allows users to pass in calling endpoint name
+	ContextEndpoint string = "endpoint"
+    // EndpointActiveStatus marks the endpoint as active
+    EndpointActiveStatus bool = true
```

Comment:
Do these vars really need to be exported? They're not used anywhere else, right?

Reply:
Nope, fixed!